gpg/gpg.go

```diff
@@ -297,6 +307,9 @@ func executeInShell(command ...string) error {
 }
 
 func executeInShellAndGrabOutput(command ...string) ([]byte, error) {
+	if keyring != "" {
+		command = append(command, "--no-default-keyring", "--keyring", keyring)
```

Comment:
any chance you can move this out of executeInShellAndGrabOutput?

Reply:
yes I guess so

Reply:
Ok thats cleaner and general